standalone-metastore/metastore-common/src/main/java/org/apache/hadoop/hive/metastore/HiveMetaStoreClient.java

```diff
@@ -4664,6 +4664,58 @@ public void dropStoredProcedure(StoredProcedureRequest request) throws MetaExcep
     return client.get_all_stored_procedures(request);
   }
 
+  /**
+   * Builder for the GetProjectionsSpec. This is a projection specification for partitions returned from the HMS.
+   */
+  public static class GetPartitionProjectionsSpecBuilder {
+
+    java.util.List<java.lang.String> partitionList = null;
+    java.lang.String includePartitionPattern = null;
+    java.lang.String excludePartitionPattern = null;
+
+    public void setPartitionList(List<String> partitionList) {
+      this.partitionList = partitionList;
+    }
+
+    public void setIncludePartitionPattern(String includePartitionPattern) {
+      this.includePartitionPattern = includePartitionPattern;
+    }
+
+    public void setExcludePartitionPattern(String excludePartitionPattern) {
+      this.excludePartitionPattern = excludePartitionPattern;
+    }
+
+    public GetProjectionsSpec build() {
+      return new GetProjectionsSpec(partitionList, includePartitionPattern, excludePartitionPattern);
+    }
+  }
+
+  /**
+   * Builder for the GetProjectionsSpec. This is a projection specification for tables returned from the HMS.
+   */
+  public static class GetTableProjectionsSpecBuilder {
+
+    java.util.List<java.lang.String> columnList = null;
```

Comment:
nit, no need to use fully qualified List here. also for String. Same for the earlier builder class as well. Please make these fields private.

Reply:
will do. Thank you for pointing these out.